include/flexflow/graph.h

```diff
@@ -335,6 +380,21 @@ struct GraphOptimizeResult {
   friend std::ostream &operator<<(std::ostream &, GraphOptimizeResult const &);
 };
 
+/**
+ * @brief Experimental. Hold the optimization results with memory information.
+ * To be merged with GraphOptimizeResult.
```

Comment:
"To be merged" when? Feel free to either include inline or (even better) link to a github issue to track this

Reply:
https://github.com/flexflow/FlexFlow/issues/620